.eslintrc.json

```diff
@@ -18,5 +18,9 @@
     "es2023": true,
     "browser": true,
     "jest/globals": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 2023,
+    "sourceType": "module"
```

Comment:
I am curious why this was added?

Reply:
I had a problem that import of module was not possible and to resolve it I have found this solution.

Reply:
removed commit b74dcfc8b9e800c1e4ae5dba610c6e36a20f4c20